index/generator/library/util_test.go

```diff
@@ -45,7 +45,19 @@ func TestCloneRemoteStack(t *testing.T) {
 			"",
 		},
 		{
-			"Case 3: Maven Java - Cloning with Hash Revision",
+			"Case 3: Wildfly Java - microprofile-config subdirectory",
```

Comment:
I know that you're only updating test cases here (plus adding a new one), but can you change it so that each test case includes the parameter names like so:

```
		{
			name: "Case 1: Maven Java",
			git: &schema.Git{
				Url:        "https://github.com/odo-devfiles/springboot-ex.git",
				RemoteName: "origin",
			},
			path: filepath.Join(os.TempDir(), "springboot-ex"),
			wantErr: false,
			wantErrStr: "",
		},
```
		
It makes it a lot easier to understand each test case

Reply:
> I'm a little confused about what will be in the cached zip:
> 
> Will the cached starter project be (folders included):
> 
> ```
> quickstart/microprofile-config
> ```
> 
> Or just
> 
> ```
> microprofile-config
> ```
It will be in the end `quickstart/microprofile-config`, the process for downloading git goes through a multi-step process before archiving. In the last step, it appears that it is archiving the entire `quickstart` but it is the contents of `quickstart/microprofile` which have been separated.